modules/service/src/main/scala/ch/epfl/bluebrain/nexus/kg/service/directives/QueryDirectives.scala

```diff
@@ -47,13 +47,26 @@ trait QueryDirectives {
     }
 
   /**
-    * Extracts the pagination and filter query params from the request or defaults to preconfigured values.
+    * Extracts the ''term'' query param from the request.
+    */
+  def term: Directive1[Option[String]] =
+    parameter('term.?).flatMap(provide(_))
```

Comment:
@MFSY do you have any thoughts on whether changing this query parameter is a good idea or not?
This used to be `q`.

Reply:
I would recommend to keep using q as query parameter just like google.com (www.google.com?q=nexus) .

Reply:
We keep it as `q` then.